libnmstate/state.py

```diff
@@ -474,6 +475,13 @@ def _remove_iface_ipv6_link_local_addr(self):
                 )
             )
 
+    def _remove_ip_stack_if_disabled(self):
+        for ifstate in six.viewvalues(self.interfaces):
```

Comment:
nit: no need for six anymore. Not critical, we need to take all these out anyway.